defender/downloaded/Contract_cToken.js

```diff
@@ -130,7 +133,7 @@ async function getTokenInfo(cTokenAddress, provider) {
 
   let decimals;
   let symbol;
-  if (oddTokens.includes(underlyingTokenAddress.toLowerCase())) {
+  if (oddTokens.indexOf(underlyingTokenAddress.toLowerCase()) !== -1) {
```

Comment:
Array.includes is supported in node 6 onward. https://node.green/#ES2016-features-Array-prototype-includes and has full browser support: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes#browser_compatibility
IMO, `x.indexOf() !== -1` seems a little hacky and less readable than `includes`

Reply:
changed to use `includes`